packages/@sanity/block-tools/package.json

```diff
@@ -49,17 +49,17 @@
     "watch": "pkg-utils watch"
   },
   "dependencies": {
+    "@sanity/types": "3.52.2",
```

Comment:
Having `@sanity/types` as a regular dependency is a pattern already used by `@sanity/migrate`, `@sanity/schema`, `@sanity/util`, and `sanity` itself. And is the preferred way of declaring them. Declaring them in `devDependencies` causes type asserters like `isPortableTextTextBlock` to be inlined, and ultimately duplicated in the resulting Studio bundle.